std/algorithm/iteration.d

```diff
@@ -2029,9 +2029,11 @@ version(none) // This requires support for non-equivalence relations
 /**
 Lazily joins a range of ranges with a separator. The separator itself
 is a range. If a separator is not provided, then the ranges are
-joined directly without anything in between them (often called $(D flatten)
+joined directly without anything in between them (often called `flatten`
 in other languages).
 
```

Comment:
I recommend setting your GH diff view to "Split":

![image](https://cloud.githubusercontent.com/assets/4370550/21446280/70d7dcf6-c8c4-11e6-9657-3ea1d7b3373f.png)